src/components/Status/Status.styles.ts

```diff
@@ -0,0 +1,196 @@
+import { StatusAppearance } from './StatusAppearance.type';
+
+import { BaseProps } from '@/types';
+
+export type StatusConfig = {
+  appearance?: Partial<
+    Record<
+      StatusAppearance,
+      { emphasis?: Partial<Record<'high' | 'medium' | 'low', BaseProps>> }
+    >
+  >;
+  dot: {
+    appearance: Partial<
+      Record<
+        StatusAppearance,
+        { emphasis?: Partial<Record<'high' | 'medium' | 'low', BaseProps>> }
+      >
+    >;
+  } & BaseProps<'appearance'>;
+  hasLabel?: BaseProps;
+  innerElements: {
+    label?: BaseProps;
+  };
+} & BaseProps<'appearance'>;
+
+export const defaultConfig = {
+  display: 'flex',
+  w: 'fit-content',
+  h: '$size-xSmall',
+  gap: '$space-component-gap-small',
+  p: '$space-component-padding-null $space-component-padding-medium',
```

Comment:
in high and medium emphasis the horizontal padding is set to `$space-component-padding-small` not `medium`, and unfortunately in low emphasis there is no padding, so you have to include that in the config object. Similarly there is no borderRadius in low emphasis state

Reply:
changed - thanks for catching that!